watertap/property_models/ion_DSPMDE_prop_pack.py

```diff
@@ -1418,6 +1524,22 @@ def calculate_scaling_factors(self):
                     ) / iscale.get_scaling_factor(self.mw_comp[j])
                     iscale.set_scaling_factor(self.conc_mol_phase_comp["Liq", j], sf)
 
+        if self.is_property_constructed("conc_equiv_phase_comp"):
+            for j in self.params.ion_set | self.params.solute_set:
+                if (
+                    iscale.get_scaling_factor(self.conc_equiv_phase_comp["Liq", j])
+                    is None
+                ):
+                    if self.is_property_constructed("charge_comp"):
+                        sf = iscale.get_scaling_factor(
+                            self.conc_mol_phase_comp["Liq", j]
+                        ) * iscale.get_scaling_factor(self.charge_comp[j])
```

Comment:
So I think if `conc_equiv` is constructed, "charge" must be constructed as part of the method.  Along with my last comment on the unnecessary scaling of "charge",  I suggest this if statement be removed.  I don't think adding a coefficient of 1/charge (usually 1, 2 or 3)  significantly affect sf carried by the conc or trigger any differences for the solver.